README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
Just checked: The script I use to setup the env and build it is:
```
...
function doPrepare {
    DEBPKGS='gnupg2 git ant openjdk-7-jdk  ghc libghc-haxml-dev libghc-tar-dev libghc-glade-dev libghc-http-dev libghc-warp-dev libghc-wai-extra-dev libghc-fgl-dev libghc-xml-dev texlive-latex-base texlive-latex-extra texlive-fonts-recommended'
    (( ${LSB_RELEASE//.} < 1510 )) && DEBPKGS+=' libghc-haskeline-dev'
    # from ppa.launchpad.net/hets
    DEBPKGS+=' libghc-aterm-dev libghc-hexpat-dev libghc-programatica-dev libghc-uni-htk-dev libghc-uni-graphs-dev libghc-uni-udrawgraph-dev'
    # zwecks tests
    DEBPKGS+=' darwin spass'    # NOTE: have indirect dep on eprove
    # für HTML API doc
    #DEBPKGS+=' libghc-haddock-dev libghc-missingh-dev'

    prepareRoot4build

    ${TAR} xJf ${SRC}/${HETS}.tar.xz
    cd ${TMPBUILD} || exit 1

    for p in ~(N)${SDIR}/patches/*.patch ; do
        ${PATCH} -p1 -E -b -z .orig -i $p
    done
    # avoid not yet ready meta-packages
    sed -i.orig -e '/TBD/,$ d' debian/control
}

function doMake {
    cd ${TMPBUILD} || exit 1

    checkMakeVars
    Log.info "make log goes to ${MAKELOG}"
    rm -rf debian/root ../hets-{common,desktop,server}_${RELEASE}_*.deb
    fakeroot make binary 2>&1 1>${MAKELOG} |tee /tmp/stderr.log

    [[ -d tmp ]] || mkdir tmp
    mv ../hets-{common,desktop,server}_${RELEASE}_*.deb tmp/
}

function doPackage {
...
```

and I guess, I see the reason: IIRC haskeline requires terminfo in ubuntu < 15.10, only ...